examples/gitlab/README.md

```diff
@@ -0,0 +1,60 @@
+Gitlab Example
+--------------
+
+### [Example Reference](https://github.com/kubernetes/charts/tree/master/stable/gitlab-ce)
+
+### Generating artifacts
+
+```
+$ kedge generate -f gitlab.yml -f redis.yml -f postgres.yml
```

Comment:
you can also give 

```
kedge create -f .
```
dot being current directory so it will take all the YAML files